ros/src/styx/bridge.py

```diff
@@ -79,7 +77,8 @@ def create_pose(self, x, y, z, yaw=0.):
         pose.pose.position.y = y
         pose.pose.position.z = z
 
-        q = tf.transformations.quaternion_from_euler(0., 0., math.pi * yaw/180.)
+        q = tf.transformations.quaternion_from_euler(
```

Comment:
what is the line length that you target? 80 chars?

Reply:
Usually I aim for 120. I think I forgot to mention the --max-linenum while doing the autopep8.